src/Findologic/Request/Handler/FilterHandler.php

```diff
@@ -174,4 +178,19 @@ private function isMaxRangeSlider(string $name): bool
     {
         return mb_substr($name, 0, mb_strlen(self::MAX_PREFIX)) === self::MAX_PREFIX;
     }
+
+    private function isPropertyFilter(string $filterName, string $filterValue): bool
+    {
+        return mb_strpos($filterValue, sprintf('%s%s', $filterName, FilterValue::DELIMITER)) === 0;
+    }
+
+    private function handlePropertyFilter(
+        string $filterName,
+        string $filterValue,
+        SearchNavigationRequest $searchNavigationRequest
+    ): void {
+        $parsedFilterValue = explode(sprintf('%s%s', $filterName, FilterValue::DELIMITER), $filterValue);
```

Comment:
* What happens if a filter name is `Best|Filter|Name`?
* What happens if a filter value is `Best|Filter|Value`?

Does Shopware prevent this?

Reply:
yes that would cause some problems I guess. Although am not sure if shopware allows that or not yet.

Reply:
But I guess that is not related to story, its a general logical issue I think. Because shopware also parses multiple values with `|` so it could be considered a bug in shopware if it allows a filter name to be that way. We might have to parse the values otherwise. We can discuss this in tomorrow's call.